Library/Formula/metashell.rb

```diff
@@ -10,20 +11,28 @@ class Metashell < Formula
     sha1 "76874d93ced1815ccfcbddb6da75e8c56aff44e4" => :mountain_lion
   end
 
-  depends_on "cmake" => :build
-
   needs :cxx11
 
+  # This patch is required because Mountain Lion uses an old compiler which breaks
+  # compiling some Templight code. The patch comments out unused parts of Templight,
+  # so patched version is functionally equivalent. This error should be fixed in
+  # the next release of Metashell.
+  patch :DATA if MacOS.version == :mountain_lion
```

Comment:
I just opened an issue for this for Metashell: https://github.com/sabel83/metashell/issues/28
@ortho please include this link in the description. Thanks!

I'll take a look a closer look why the build failure happened before we release the next version of Metashell.